src/Stryker.Core/Stryker.Core.UnitTest/Reporters/ReportTestHelper.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Stryker.Core.UnitTest.Reporters
 {
-    public static class JsonReportTestHelper
+    public static class ReportTestHelper
```

Comment:
This rename might be controversial (happy to put it back) I just noticed it was being used in a few places to generate test report data, not just for the Json Report.

Reply:
This is fine with me